examples/hitl/basic_viewer/basic_viewer.py

```diff
@@ -113,27 +114,35 @@ def _get_controls_text(self):
         controls_str += "ESC: exit\n"
         if self._episode_helper.next_episode_exists():
             controls_str += "M: next episode\n"
-
+        else:
+            controls_str += "no remaining episodes\n"
+        controls_str += "P: unpause\n" if self._paused else "P: pause\n"
+        controls_str += "Spacebar: single step\n"
+        controls_str += "R + drag: rotate camera\n"
+        controls_str += "Scroll: zoom\n"
         controls_str += "I, K: look up, down\n"
-        controls_str += "A, D: look left, right\n"
-        controls_str += "O, P: move up, down\n"
-        controls_str += "W, S: move forward, backward\n"
+        controls_str += "A, D: turn\n"
+        controls_str += "E, Q: move up, down\n"
+        controls_str += "W, S: move forward, back\n"
 
         return controls_str
 
     def _get_status_text(self):
-        status_str = ""
-
+        progress_str = f"episode {self._sandbox_service.episode_helper.current_episode.episode_id}"
         if not self._env_episode_active():
-            if self._env_task_complete:
-                status_str += "Task complete!\n"
-            else:
-                status_str += "Oops! Something went wrong.\n"
+            progress_str += (
```

Comment:
This help text polish is unrelated to the Hydra config migration, but I needed this to properly test that I'd ported the episodes_filter config param. Apologies for the confusion.